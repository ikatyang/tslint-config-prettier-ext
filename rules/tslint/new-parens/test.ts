class X {}
const x = new X;
