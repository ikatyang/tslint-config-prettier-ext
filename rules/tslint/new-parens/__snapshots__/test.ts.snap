// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (new-parens) /rules/tslint/new-parens/test.ts[2, 11]: Parentheses are required when invoking a constructor

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
class X {}
const x = new X;

======
class X {}
const x = new X();

>>>>>> after
"
`;
